galite-core/src/main/kotlin/org/kopi/galite/ui/vaadin/notif/VInformationNotification.kt

```diff
@@ -18,66 +18,37 @@
 package org.kopi.galite.ui.vaadin.notif
 
 import org.kopi.galite.ui.vaadin.base.LocalizedProperties
-import org.kopi.galite.ui.vaadin.base.VInputButton
 
-import com.vaadin.flow.component.ShortcutEvent
-import com.vaadin.flow.component.html.H3
-import com.vaadin.flow.component.html.Span
-import com.vaadin.flow.component.orderedlayout.FlexComponent
-import com.vaadin.flow.component.orderedlayout.HorizontalLayout
+import com.vaadin.flow.component.button.Button
+import com.vaadin.flow.component.icon.VaadinIcon
 
 /**
  * Information type notification component.
+ *
+ * @param title the information notification title.
+ * @param message the information notification message.
```

Comment:
can we add locale in the doc of VInformationNotification, VConfirmNotification, VErrorNotification & VWarningNotification